### PR TITLE
feat(RHTAPWATCH-1038): Support custom certificate in ec pipeline

### PR DESCRIPTION
**Pull request by hmariset**
Add ConfigMap parameters to support custom certificate in enterprise-contract pipeline.

Jira-Url: https://issues.redhat.com/browse/RHTAPWATCH-1038

# Before you complete this pull request ...

Look for any open pull requests in the repository with the title "e2e-tests update" and 
see if there are recent e2e-tests updates that will be applicable to your change.

### CODE REVIEW

File: pipelines/enterprise-contract.yaml [hunk 1]
```diff
@@ -75,6 +83,10 @@ spec:
           value: "true"
         - name: TIMEOUT
           value: "$(params.TIMEOUT)"
+        - name: CA_TRUST_CONFIGMAP_NAME
```
Comment (lcarva) [inline pipelines/enterprise-contract.yaml:86]: The `ec-task-bundle` needs to be updated to include a version of the EC Task that accepts these parameters. The current reference does not:

```bash
🐚 tkn bundle list -o yaml quay.io/enterprise-contract/ec-task-bundle:snapshot@sha256:294b14582fa0e44f42c7e0651915ca67425488527fa7d9ecb49c3974ede028fc | yq .spec.params[].name
*Warning*: This is an experimental command, it's usage and behavior can change in the next release(s)
DEFINITION
POLICY_SOURCE
NAMESPACE
POLICY_LIB
POLICY_DATA
HOMEDIR
```

Comment (yftacherzog) [reply pipelines/enterprise-contract.yaml:86]: @lcarva If I understand correctly, that command returns the `.spec.params[].name` of the last spec it finds, which is the spec of the `verify-definition` task.
I don't think this is actually a valid yaml so yq might not be the tool to process it.

Looking at the output, without yq, I do see those parameters under the `verify-enterprise-contract` task.

File: pipelines/enterprise-contract.yaml [hunk 2]
```diff
@@ -51,6 +51,14 @@ spec:
       type: string
       description: Timeout setting for `ec validate`.
       default: "5m0s"
+    - name: CA_TRUST_CONFIGMAP_NAME
+      type: string
+      description: The name of the ConfigMap to read CA bundle data from.
+      default: "trusted-ca"
```
[the remark this answers is not in the document]
Comment (yftacherzog) [reply pipelines/enterprise-contract.yaml:57]: Not necessarily, but it's optional on the task, so if it doesn't exist on the namespace, then the task should not hang. The volume mount creates a file that does not exist on the container, so it should not conflict with existing certificates.

[the remark this answers is not in the document]
Comment (lcarva) [reply pipelines/enterprise-contract.yaml:57]: Perfect. Thank you!

File: pipelines/enterprise-contract.yaml (hunk 1 above)
Comment (hmariset) [reply pipelines/enterprise-contract.yaml:86]: Thanks @lcarva and @yftacherzog!

Comment (lcarva) [reply pipelines/enterprise-contract.yaml:86]: FYI https://github.com/tektoncd/cli/issues/2091

**Pull request merged**